babel/core.py

```diff
@@ -885,6 +885,41 @@ def ordinal_form(self):
         """
         return self._data.get('ordinal_form', _default_plural_rule)
 
+    @property
+    def measurement_systems(self):
+        """Localized names for various measurement systems.
+
+        >>> Locale('fr', 'FR').measurement_systems['US']
+        u'am\\xe9ricain'
+        >>> Locale('en', 'US').measurement_systems['US']
+        u'US'
+
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
+        """The text direction for the language.
+
+        >>> Locale('de', 'DE').character_order
+        'left-to-right'
+        >>> Locale('ar', 'SA').character_order
+        'right-to-left'
+        """
+        return self._data['character_order']
+
+    @property
+    def direction(self):
+        """The text direction for the language in CSS short-hand form.
+
+        >>> Locale('de', 'DE').direction
+        'ltr'
+        >>> Locale('ar', 'SA').direction
+        'rtl'
+        """
+        order = self.character_order
+        return ''.join([word[0] for word in order.split('-')])
```

Comment:
What would you like to see unit tested?

We know empirically (`grep | sort | uniq`) `left-to-right` and `right-to-left` are the only `character_order`s we'll ever see, so `ltr` and `rtl` will also be the only possible `direction`s. (And this is future-proof: if we ever get `top-to-bottom` or `bottom-to-top` or `upside-down-and-dinky-donky`, those will be handled too.)